app/internal/page/certificateprovider/mock_Template_test.go

```diff
@@ -0,0 +1,43 @@
+// Code generated by mockery v2.20.0. DO NOT EDIT.
+
+package certificateprovider
+
+import (
+	io "io"
+
+	mock "github.com/stretchr/testify/mock"
+)
+
+// mockTemplate is an autogenerated mock type for the Template type
+type mockTemplate struct {
+	mock.Mock
+}
+
+// Execute provides a mock function with given fields: _a0, _a1
+func (_m *mockTemplate) Execute(_a0 io.Writer, _a1 interface{}) error {
```

Comment:
Is this what is eventually being called when `template.Func` is run in our handlers?

Reply:
yep, it works just like `Func` did before